modules/cardano-node-aws.nix

```diff
@@ -4,16 +4,17 @@ testIndex: region:
   { config, resources, pkgs, nodes, ... }:
     let
       cfg = config.services.cardano-node;
+      cardanoNodes = filter (c: c.services.cardano-node.enable)
```

Comment:
I would have tried to find a more descriptive name for this,
like `enabledNodeConfigs`, or somesuch..